lib/requirejs/require.js

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
You’re totally right. I said something stupid.

I’ve remove the RequireJS code with 49b17a2, getting rid of the lib directory (hurray!).